store/src/java/com/zimbra/cs/account/Provisioning.java

```diff
@@ -1048,6 +1047,14 @@ public abstract Account restoreAccount(String emailAddress, String password,
      */
     public abstract void deleteAccount(String zimbraId) throws ServiceException;
 
+    /**
+     * Checking whether the license which is available in the server is valid or not
+     * @throws ServiceException
```

Comment:
```suggestion
     * @throws ServiceException if license is not valid, else return void
```

Reply:
This code has now been removed due to changes in logic.